python/cudf/cudf/tests/test_index.py

```diff
@@ -2795,3 +2795,21 @@ def test_index_error_list_index():
         ),
     ):
         cudf.Index(s)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [1, 2, 3],
+        [np.nan, 10, 15, 16],
+        range(0, 10),
+        [np.nan, None, 10, 20],
+        ["ab", "zx", "pq"],
+        ["ab", "zx", None, "pq"],
+    ],
+)
+def test_index_hasnans(data):
+    gs = cudf.Index(data, nan_as_null=False)
+    ps = gs.to_pandas()
```

Comment:
Why does the other test use `nullable=True` but not this one?

Reply:
We are yet to add this support of creating a nullable dtypes in pandas Index: https://github.com/rapidsai/cudf/issues/12216

Reply:
Can you add a note in that issue to follow up and fix this?